test/services/katello/pulp/repository/yum_test.rb

```diff
@@ -210,6 +210,17 @@ def test_create_with_global_http_proxy
           assert_equal "http://" + uri.host, config['proxy_host']
         end
 
+        def test_create_with_global_http_proxy_ipv6
+          @default_proxy = FactoryBot.create(:http_proxy, name: 'best proxy', url: "http://[2001:db8:abcd::1]")
```

Comment:
test failure indicates a proxy with this name already exists:
ActiveRecord::RecordInvalid: Validation failed: Name has already been taken

Reply:
Yeah sorry got busy. Blindly rebased with fingers crossed :-)